src/L1/rollup/ScrollChain.sol

```diff
@@ -713,18 +907,22 @@ contract ScrollChain is OwnableUpgradeable, PausableUpgradeable, IScrollChain {
         uint256 _length;
         if (version == 0) {
```

Comment:
We keep this because `importGenesisBatch` uses it? raise this question because, in commitBatch, version 0 is deprecated.

Reply:
Yes